packages/genesys-spark-components/src/components/stable/gux-pagination-cursor/gux-pagination-cursor.tsx

```diff
@@ -76,6 +87,7 @@ export class GuxPaginationCursor {
             type="button"
             disabled={!this.hasPrevious}
             onClick={() => this.onButtonClick('previous')}
+            aria-controls={this.controls}
```

Comment:
I would be surprised if this works as the ids are not in the same context. How have you confirmed it as working?

Is
```
<gux-pagination-cursor aria-controls="default"></gux-pagination-cursor>
```
not enough?

We may have to look into slotting buttons so they are in the light dom for accessibility.